nrf_rpc/include/nrf_rpc_common.h

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _NRF_RPC_COMMON_H_
+#define _NRF_RPC_COMMON_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup nrf_rpc_common Common nRF RPC utility
```

Comment:
OK, but if you are planning to move this **later**, so why we should change the naming right now? In my opinion, it is better to change the naming to be more common when we actually make this file common.

Reply:
Because then there is only the matter of moving the header file, and the names would not need to be updated.

The reason why I don't suggest to move it now is because there is infrastructure missing for common headers in nrfxlib, which is why we don't have a common nrf_errno.h yet.

Reply:
I agree that definitions contained in this file may be useful commonly.
Trying to make it common creates some questions that are not really related to this PR, e.g.:
* Does the entire file should be made common? If not which parts of it should?
* What prefix should be given to the names? `NRFXLIB_`, `NRF_`, none or some other?
* What compilers should it support?

Answering those questions is required to do the changes that you suggesting. If you have clear picture how this file should look like in "common" version then I can change it, but for me it is not clear.

On the other hand, if we postpone this job to some more general task (that also includes e.g. moving this file and creating common nrf_errno.h" then renaming is pretty easy job. Simple "Find and replace" will work, especially when we have unique "nrf_rpc_" prefix in names.

So, my question is: Can you provide more details how this file should look like or should we postpone renaming for later?

Reply:
@joerchan Can you provide more details on changes that you suggesting or should we postpone this sometime after PR merge?